src/pages/ForgotPassword/ForgotPassword.js

```diff
@@ -43,8 +43,11 @@ export default function ForgotPassword(props) {
             validator={(value) => value && isEmail(value)}
             icon="person-outline"
           />
-          <button className="submit-button" type="submit">
-            Send
+          <button className="arrow-icon" type="submit">
+            <ion-icon
+            name="arrow-forward-circle-outline"
+            style={{fontSize: "40px", "--ionicon-stroke-width": "40px"}}
+            ></ion-icon>
```

Comment:
To make this accessible, I'd recommend still including words like "login" and "sign out". You can put the icon inside the button inlined with the description. If you prefer the minimalism aesthetically, probably a good idea to stick an `aria-label` on there in case one of our members uses a screen reader.

Reply:
Great idea, check out 4b45900 and see if this addresses your concerns! Dealing with accessibility is new to me so open to any further feedback. Personally I do prefer the minimalism of the arrow buttons, so if we can make it work I'd like to stick with that :)